Makefile

```diff
@@ -1,158 +1,7 @@
-#
```

Comment:
I think this Makefile is no longer used and we use `quisp/Makefile`.
so I remove this file and put new Makefile for clang-format.